internal/integration/testdata/mysql/column-add-drop-2.txt

```diff
@@ -0,0 +1,110 @@
+# Each test runs on a clean database.
+
+# Apply schema "1.hcl" on fresh database.
+apply 1.hcl
+
+# Ensures tables exist in the database.
+exist users
+
+# Compare the result of "SHOW TABLE users" with the content of a file named '1.sql'.
+# The "cmpshow" command searches first a file named '<version>/1.sql' (version, defines
+# the database version), and in case it was found, it will use it instead.
+cmpshow users 1.sql
+
+# Apply schema "2.hcl" on the updated database.
+apply 2.hcl
+
+# Compare database with 2.sql.
+cmpshow users 2.sql
+
+# Apply schema "1.hcl" should migrate database to previous state.
+apply 1.hcl
+cmpshow users 1.sql
+
+# Drop table.
+apply 0.hcl
+! exist users
```

Comment:
Add another assertion using `cmphcl` to ensure HCL writing is correct. Also, no need for another test, you can extend the current one.

Reply:
You can extend this one: https://github.com/ariga/atlas/blob/master/internal/integration/testdata/mysql/column-generated-inspect.txt#L5